learningresources/api.py

```diff
@@ -120,13 +103,93 @@ def type_id_by_name(name):
     return obj.id
 
 
-def get_repos(user):
+def get_repos(user_id):
     """
     Get all repositories a user may see.
 
     Args:
         user (auth.User): request.user
     Returns:
-        repos query set of learningobject.Repository: repositories
+        repos query set of learningresource.Repository: repositories
+    """
+    return Repository.objects.filter(created_by__id=user_id).order_by('name')
+
+
+def get_repo_courses(repo_id):
+    """
+    Get courses for a repository.
+    Args:
+        repository (int): pk of learningresource.Repository
```

Comment:
Docstring needs update (sorry for all these messages, trying to make it easier to spot these)